Code/application/PlugInUtilities/MatrixFunctions.cpp

```diff
@@ -364,3 +366,67 @@ bool MatrixFunctions::solveLinearEquation(double* pResult, const double** pLhs,
    memcpy(pResult, pData, resultVector.Storage() * sizeof(pData[0]));
    return true;
 }
+#else
+// ossim 1.9.0 doesn't have an explicit SVDSolve() function, so we'll use Eigen's
```

Comment:
There are a number of changes that were made to ossim that are not in their trunk (opticks runs off a fork) which is why there is no SVDSolve() available in stock 1.9.0. I would like to get the ossim dependency out of the required build and limit it to builds requiring NITF. This is a good first step (eigen wasn't available when this code was first written).

If you see any other changes like this, they can go into master as well. Once I have most things up to date and working with clean dependencies I intend to go over all the other ossim changes and either remove the need for them or push the changes to the ossim project.